### DISCUSSION TITLE
Website link broken?

### DISCUSSION THREAD

**Discussion by mramirez-trimac** [JustFly1984/react-google-maps-api#3338] [General]
Hello! 
When I try to access the documentation / website for this library I'm seeing a "site not found" error. Is this happening to anyone else? Is there another website where I can find the documentation for this library? 

![image](https://github.com/JustFly1984/react-google-maps-api/assets/102977437/b7277050-72e1-4a9b-ae37-19e792dc44a2)

The link I'm trying to use is [this one](https://react-google-maps-api-docs.netlify.app/), which I found on the [npm site](https://www.npmjs.com/package/@react-google-maps/api) 

Thanks in advance for your help :)

**Comment by kevinjavitz** [comment]
Same for me, can you fix the docs and examples website?
